### ISSUE TITLE
Got build error when enabling shrink without proguard

### ISSUE THREAD

**Issue by zaki50** [realm/realm-java#3714]
#### Goal

Successfully build an app with shrinking but without ProGuard
#### Expected Results

no error on build.
#### Actual Results

```
:app:transformClassesWithNewClassShrinkerForRelease FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:transformClassesWithNewClassShrinkerForRelease'.
> ProGuard configuration parser error: /Users/zaki/fromgit/realm/RealmTasks/RealmTasks Android/app/build/intermediates/exploded-aar/io.realm/realm-android-library-object-server/2.1.0/proguard.txt line 5:6 no viable alternative at input 'includedescriptorclasses'
```
#### Steps & Code to Reproduce

Add following configuration in `app/build.gradle`

```
android {
    buildTypes{
        release {
            minifyEnabled true // add
            useProguard false  // add
        }
    }
}
```
#### Version of Realm and tooling

Realm version(s): 2.1.1

Realm sync feature enabled: both

Android Studio version: 2.2.2

Which Android version and device: 7.1.1

**Comment by zaki50** [comment]
`includedescriptorclasses` is not supported by built-in shrinker http://tools.android.com/tech-docs/new-build-system/built-in-shrinker

**Comment by Zhuinden** [comment]
Out of curiosity, is there a way to "opt out" of the Proguard file that Realm's plugin gives you?

**Comment by zaki50** [comment]
@Zhuinden No idea...

**Comment by zaki50** [comment]
I've created an issue on b.android.com

https://code.google.com/p/android/issues/detail?id=226531

**Comment by zaki50** [comment]
fixed in `2.2.0`
